content/2.2database.md

```diff
@@ -221,4 +221,4 @@ the language/library takes care to identify the exception type for us and return
 #### Links
 
 - Previous section: [Basic Functionality](content/2.1functionality.md)
-- Next section: 
\ No newline at end of file
+- Next section: 
```

Comment:
This final line edit and other other ones in the pull requests I made just now are all artifacts of the github built-in editor. I apologize for this.